apps/console/src/features/users/components/wizard/add-user-wizard.tsx

```diff
@@ -744,12 +972,392 @@ export const AddUserWizard: FunctionComponent<AddUserWizardPropsInterface> = (
+     */
+    const getSummaryWizardStep = (): WizardStepInterface => {
+
+        return {
+            content: (
+                <AddConsumerUserWizardSummary
+                    triggerSubmit={ finishSubmit }
+                    onSubmit={ handleWizardFormFinish }
+                    summary={ generateWizardSummary() }
+                    onProfileImageChange={ handleProfileImageChange }
+                />
+            ),
+            icon: getUserWizardStepIcons().summary,
+            name: WizardStepsFormTypes.SUMMARY,
+            title: t("console:manage.features.user.modals.addUserWizard.steps.summary")
+        };
+    };
+
+
+    /**
+     * User summary wizard step.
+     * @returns User summary wizard step.
+     */
+    const getUserSummaryWizardStep = (): WizardStepInterface => {       
+         
+        return {
+            content: (
+                <AddUserWizardSummary
+                    triggerSubmit={ finishSubmit }
+                    selectedUserStore = { selectedUserStore }
+                    username={ 
+                        isAlphanumericUsername
+                            ? wizardState[ WizardStepsFormTypes.BASIC_DETAILS ]?.userName
+                            : wizardState[ WizardStepsFormTypes.BASIC_DETAILS ]?.email
+                    }
+                    password={ wizardState[ WizardStepsFormTypes.BASIC_DETAILS ]?.newPassword }
+                    isPasswordBased={ askPasswordFromUser }
+                />
+            ),
+            icon: getUserWizardStepIcons().summary,
+            name: WizardStepsFormTypes.USER_SUMMARY,
+            title: t("console:manage.features.user.modals.addUserWizard.steps.summary")
+        };
+    };
+
+        /**
+     * This function handles sending the invitation to the external admin user.
+     */
+        const sendParentOrgInvitation = (invite: UserInviteInterface) => {
+            debugger;
+            if (invite != null) {
+                setIsSubmitting(true);
+    
+                sendParentOrgUserInvite(invite)
+                    .then(() => {
+                        dispatch(addAlert({
+                            description: t(
+                                "console:manage.features.invite.notifications.sendInvite.success.description"
+                            ),
+                            level: AlertLevels.SUCCESS,
+                            message: t(
+                                "console:manage.features.invite.notifications.sendInvite.success.message"
+                            )
+                        }));
+                        closeWizard();
+                    })
+                    .catch((error: AxiosError) => {
+                        // Axios throws a generic `Network Error` for 401 status.
+                        // As a temporary solution, a check to see if a response
+                        // is available has be used.
+                        console.log("ERROR: ", error);
```

Comment:
Remove console.log statements. Check in other places as well. There should not be any `debuggers` or `console.log()`